src/main/java/org/apache/commons/text/StringTokenizer.java

```diff
@@ -848,11 +844,9 @@ private int readWithQuotes(final char[] srcChars, final int start, final int len
                     pos += trimmedLen;
                     continue;
                 }
-
-                // copy regular character from outside quotes
-                workArea.append(srcChars[pos++]);
-                trimStart = workArea.size();
             }
+            workArea.append(srcChars[pos++]);
```

Comment:
Ditto here about the comment that was removed :+1:

Reply:
changed. TY